Lib/test/test_peg_generator/ast_dump.py

```diff
@@ -0,0 +1,60 @@
+def ast_dump(node, annotate_fields=True, include_attributes=False, *, indent=None):
```

Comment:
Heh, if it's just this file that's duplicated, I don't mind. Sorry for making a fuss!